app/flows/check_uk_visa_flow/outcomes/outcome_marriage_electronic_visa_waiver.erb

```diff
@@ -11,5 +11,9 @@
   - settled in the UK
   - someone with humanitarian protection, for example if they have refugee status
 
-  ^If you want to convert a civil partnership into a marriage, you'll need to apply for an [electronic visa waiver (EVW)](/get-electronic-visa-waiver) or a [Standard Visitor visa](/standard-visitor).
+  If you want to convert a civil partnership into a marriage, you'll need to apply for an [electronic visa waiver (EVW)](/get-electronic-visa-waiver) or a [Standard Visitor visa](/standard-visitor).
+
+  <%if calculator.passport_country_requires_electronic_travel_authorisation?  %>
```

Comment:
Surprised rubocop hasn't flagged this, but the house-style is `<% if ... %>` - so one space after `%` and one before.

Reply:
We will need [erb-lint](https://github.com/Shopify/erb-lint) for it which I can add in a separate PR. For now, will add the space manually 👍